code/src/Components/Details.css

```diff
@@ -0,0 +1,111 @@
+ .movie-overview {
```

Comment:
Code organization: The code is well-organized and easy to read, with clear separation between different sections and components of the page. This makes it easy to maintain and update the code in the future